win32ss/user/user32/windows/text.c

```diff
@@ -164,13 +175,21 @@ LPSTR
 WINAPI
 CharPrevExA(WORD codepage, LPCSTR start, LPCSTR ptr, DWORD flags)
 {
-    while (*start && (start < ptr))
+    if (ptr > start)
     {
-        LPCSTR next = CharNextExA(codepage, start, flags);
-        if (next >= ptr) break;
-        start = next;
+        BOOL dbl = FALSE;
+
+        --ptr;
+        for (LPCSTR ch = ptr - 1; ch >= start; --ch)
```

Comment:
```suggestion
        LPCSTR ch:
        --ptr;
        for (ch = ptr - 1; ch >= start; --ch)
```

Reply:
There are no checks against `flags` or against `gpsi->dwSRVIFlags` here?

Reply:
Yes, the second function doesn't check that flag.